.github/workflows/continuous-integration-workflow.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: False
       matrix:
         python-version: ["3.9", "3.10", "3.11"]
-        os: [ubuntu-latest] #, macos-latest, windows-latest]
+        os: [ubuntu-20.04] #, macos-latest, windows-latest]

```

Comment:
Strictly, it's only this line that actually had to change to have all tests pass (for some reason, the check-working-examples workflow was still running correctly). However, for consistency, I've switched all instances of `ubuntu-latest` to `ubuntu-20.04`.

Reply:
Agree with this